dpdk.spec

```diff
@@ -226,7 +202,7 @@ done
 
 %if ! %{with tools}
 rm -rf %{buildroot}%{sdkdir}/usertools
-rm -rf %{buildroot}%{_sbindir}/dpdk-devbind
+rm -rf %{buildroot}%{_bindir}/dpdk-devbind.py
```

Comment:
Why the change to `_bindir`?  This is a super-user tool isn't it?  Needs root to run?  If so, it should be in `_sbindir`